data/optimus-metrics-cache.json

```
@@ -3,5 +3,5 @@
     "latestAvgApr": -12.345452875000001,
     "latestETHApr": -8.108115914125001
```

with be more logical with the update. then once the page is visited by someone, the file is updated on Vercel side
```suggestion
    "latestAvgApr": 0.00,
    "latestETHApr": 0.00
```